src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,9 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder palindrome;
+        palindrome = new StringBuilder(text.toLowerCase().replaceAll("[^a-zA-Z0-9]", ""));
+        palindrome.reverse();
+        return text.toLowerCase().replaceAll("[^a-zA-Z0-9]", "").equals(palindrome.toString());
```

Comment:
Good job! Try to make this code shorter.

Reply:
done